activerecord/lib/active_record/railties/console_sandbox.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 ActiveRecord::ConnectionAdapters::AbstractAdapter.set_callback(:checkout, :after) do
-  begin_transaction(joinable: false)
+  begin_transaction(fixtures: true)
```

Comment:
@matthewd I had to rename it here, which perhaps is at least one good argument to keep it called as `joinable` instead of `fixtures`.

We could do that and keep `joinable` and then just provide `def fixture_transaction?` as an accessor.